tests/test_code_actions.py

```diff
@@ -0,0 +1,60 @@
+from LSP.plugin.core.protocol import Point, Range
+from LSP.plugin.core.url import filename_to_uri
+from LSP.plugin.core.views import entire_content
+from LSP.plugin.code_actions import run_code_action_or_command
+from setup import TextDocumentTestCase
+from test_single_document import TEST_FILE_PATH
+
+try:
+    from typing import Generator, Optional, Iterable, Tuple, List, Dict
+    assert Generator and Optional and Iterable and Tuple and List and Dict
+except ImportError:
+    pass
```

Comment:
Please don't use these `try import ...` idioms anymore. You can use `from LSP.plugin.core.typing import Generator, Optional, Iterable, Tuple, List, Dict` at the top of the file. You can also remove quotes from types.